### PR TITLE
Minor fix on the docs

### PR DESCRIPTION
**Pull request by decarlof**

### CODE REVIEW

**Comment by prjemian** [comment]
Have you considered a more-recent [development](http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports) that simplifies the Mock process?

Example use from the [apstools documentation](https://github.com/BCDA-APS/apstools/blob/master/docs/source/conf.py#L173-L174).

**Comment by prjemian** [comment]
perhaps changing

```
# picked from http://read-the-docs.readthedocs.org/en/latest/faq.html
class Mock(object):

    __all__ = []

    def __init__(self, *args, **kwargs):
        pass

    def __call__(self, *args, **kwargs):
        return Mock()

    @classmethod
    def __getattr__(cls, name):
        if name in ('__file__', '__path__'):
            return '/dev/null'
        elif name == 'c_byte':
            return 0
        else:
            return Mock()
    # https://docs.python.org/2/library/operator.html
    def __mul__(self, other):
        return Mock()
    def __rmul__(self, other):
        return Mock()
    def __pow__(self, other):
        return Mock()
    def __div__(self, other):
        return Mock()
    def __add__(self, other):
        return Mock()
    def __radd__(self, other):
        return Mock()
    def __or__(self, other):
        return Mock()
    
curFlags = Mock()
   
MOCK_MODULES = [
    'numpy', 'scipy', 'scipy.misc', 'scipy.optimize', 'matplotlib', 
    'matplotlib.pylab', 'tifffile', 'scipy.ndimage', 'scipy.signal', 
    'pywt', 'scikit-image', 'skimage', 'skimage.io', 'skimage.filter', 
    'skimage.morphology', 'skimage.feature', 'skimage.transform', 
    'DM3lib', 'pyfftw', 'dxchange', 'numexpr', 'concurrent', 
    'concurrent.futures', 'libtomopy', 
    ]

for mod_name in MOCK_MODULES:
sys.modules[mod_name] = Mock()
```

into

```
autodoc_mock_imports = "numpy scipy matplotlib tifffile pywt skimage DM3lib pyfftw dxchange numexpr concurrent libtomopy".split()
```

**Comment by decarlof** [comment]
Please also address this [issue](https://github.com/tomopy/tomopy/issues/401) to properly build the docs

File: doc/source/conf.py [hunk 1]
```diff
@@ -300,50 +300,6 @@
 #ztexinfo_no_detailmenu = False
 
 # picked from http://read-the-docs.readthedocs.org/en/latest/faq.html
```
Comment (prjemian) [inline doc/source/conf.py]: replace with

```
# http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports
```

Comment (decarlof) [reply doc/source/conf.py]: thanks, done

**Comment by carterbox** [comment]
@dgursoy You are the only one with admin access to TomoPy's read the docs. Thus, you have to fix the build.

**Pull request merged**